### DISCUSSION TITLE
47회차 반성문

### DISCUSSION THREAD

**Discussion by hyun98** [BDD-CLUB/challenge-every-week-posting#494] [6. 반성문]
벌써 2번째 반성문이라니;; 죄송합니다
지난주는 본가에 가서 오랜만에 가족, 친구들 본다고 글 작성을 못했네요ㅜ
그래도 오랜만에 밀면을 먹을 수 있어서 행복했습니다
아참 부산대 잔디 생기고 처음 가봤는데 너무 신기하더라구요ㅋㅋㅋ

**Comment by gusah009** [comment]
부산대에 잔디가 생겼다니
